gen_release_notes.pl

```diff
@@ -131,20 +132,27 @@
     my $dep_issues_url = "https://api.github.com/repos/$dep/issues";
     my $closed_dep_issues = get_issues($dep_issues_url, fuzz_time($from_dep_time, -5), fuzz_time($to_dep_time, 5));
 
-    push @$merged_prs, @$merged_dep_prs;
+    $merged_prs_dict{$dep} = $merged_dep_prs;
     push @$closed_issues, @$closed_dep_issues;
 }
 
-print "# Merged PRs\n\n";
-foreach my $pr (@$merged_prs) {
-    print "* [$pr->{number}]($pr->{url}): $pr->{title}\n";
+my @sorted_deps = nsort keys %merged_prs_dict;
```

Comment:
Do you need nsort here? Aren't these just strings (repo names)?

Reply:
they are strings, but im sorting them so the `dolt` changes print first, then gms, then vitess... it looked weird when vitess came before the dolt changes in the output